managed/ui/src/components/common/associatedUniverse/AssociatedUniverse.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { YBModal } from '../forms/fields';
+import { BootstrapTable, TableHeaderColumn } from 'react-bootstrap-table';
+import { Link } from 'react-router';
+import './AssociatedUniverse.scss';
+
+function AssociatedUniverse(props) {
+  const { onHide, associatedUniverses, visible, title } = props;
+  /**
+   * Returns the decorated status.
+   * @param universeStatus status of universe.
+   */
+  const modifyStatus = (item,row) => {
+    const universeStatus = row.universePaused
+      ? 'Paused'
+      : row.updateSucceeded && !row.updateInProgress
+      ? 'Ready'
+      : 'Error';
+    // TODO: Add case for pause status.
+    switch (universeStatus) {
+      case 'Ready':
+        return (
+          <div className="universe-status associated-universe-status good">
+            <i className="fa fa-check-circle associated-universe-icon" />
+            {universeStatus && <span>{universeStatus}</span>}
+          </div>
+        );
+      case 'Error':
+        return (
+          <div className="universe-status associated-universe-status bad">
+            <i className="fa fa-warning associated-universe-icon" />
+            {universeStatus && <span>{universeStatus}</span>}
+          </div>
+        );
+      case 'Paused':
+        return (
+          <div className="universe-status associated-universe-status paused">
+            <i className="fa fa-pause-circle-o associated-universe-icon" />
+            {universeStatus && <span>{universeStatus}</span>}
+          </div>
+        );
+    }
+  };
+
+  /**
+   * Returns the universe name with router link.
```

Comment:
I believe this is used to get the respective details of the particular universe. So update the doc accordingly.

Reply:
Actually, this is formatter function that will dynamically modify the universe name field to have a link.